include/singa/utils/context.h

```diff
@@ -47,7 +47,7 @@ CHECK_EQ(error, cudaSuccess) << " " << cudaGetErrorString(error); \
 #include <cudnn.h>
```

Comment:
This file is not used and will be removed soon.
If your code has dependency on it, please resolve it.